### PR TITLE
Remove `Copyright` and `Created by` from comment header

### PR DESCRIPTION
**Pull request by lukasz-szyszkowski**
There is a lot of changes but all of them is just removing the header comment lines.

### CODE REVIEW

File: Source/ARTAuth+Private.h
Comment (QuintinWillison) [inline Source/ARTAuth+Private.h:2]: These are superfluous. They also add maintenance overhead (if this file is renamed) and cognitive load for maintainers, inspectors and reviewers alike.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]
Thanks, but more work needs to be done here. Please remove the ENTIRE header blocks.

File: Source/ARTAuth+Private.h
Comment (lukasz-szyszkowski) [reply Source/ARTAuth+Private.h]: here db1308345753fbe2e41eea5d247233ad3afd2934

**Review by maratal — APPROVED** [review]

**Review by ben-xD — APPROVED** [review]
LGTM

I was wondering though, what tool did you use to autoformat `Source/ARTPushChannelSubscriptions.m`

Edit: Okay Xcode then :P. its more rudimentary compared to AppCode 😂🤓

**Comment by lukasz-szyszkowski** [comment]
> I was wondering though, what tool did you use to autoformat `Source/ARTPushChannelSubscriptions.m`

CMD+A, CTRL+I ;)

**Review by QuintinWillison — APPROVED** [review]
All those bytes saved when people `clone` this repo now. 😆 

Thanks for doing this. I know it was tedious but it keeps the house clean.

**Comment by ben-xD** [comment]
Actually because every change causes a new file to be stored in git, it should add bytes, not remove them 😅. A clone will download all files, including the old version and the new one.

In practice, I hope this is not a problem because of compression somewhere. That got me thinking, does git cloning between https vs ssh have different performance? And what the compression is like

File: Source/ARTAuth+Private.h
Comment (lukasz-szyszkowski) [reply Source/ARTAuth+Private.h]: @QuintinWillison I can't merge it without resolving this conversation - please resolve it (if it's ok) or add another comment (if it's not). Thanks

Comment (QuintinWillison) [reply Source/ARTAuth+Private.h]: Sorry for the delay in responding to this. I'm resolving this conversation now but will leave it for you to land the PR.

**Pull request merged**
